docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,398 @@
+            id='1'
+            idealDirection="up"
+            onChange={handleSetStartDate}
+
+            placeholder='Start Date'
+            selectsStart
+            value={startDate}
+          />
+        </Box>
+        <Box column={6} paddingX={1}>
+          <DatePicker
+            startDate={startDate}
+            accessibilityLabelNextMonth='Go to next month'
+            accessibilityLabelPreviousMonth='Go to previous month'
+            accessibilityLabelIcon='Select an end date'
+            endDate={endDate}
+            id='2'
+            idealDirection="up"
+            onChange={handleSetEndDate}
+
+            placeholder='End Date'
+            selectsEnd
+            value={endDate}
+          />
+        </Box>
+      </Box>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="example-disabled"
+    description="
+    DatePicker can display its own helper message for cases where you want to provide more information about the date field.
+  "
+    name="Example: Disabled"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <DatePicker
+      accessibilityLabelNextMonth='fakeaccessibilityLabelNextMonth'
+      accessibilityLabelPreviousMonth='fakeaccessibilityLabelPreviousMonth'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      disabled
+      id="disabled"
+      onChange={({value}) => setDate(value)}
+      value={date}
+    />
+
+  )
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="helperText"
+    description="
+    DatePicker can display its own helper message for cases where you want to provide more information about the date field.
+  "
+    name="Example: helperText"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <DatePicker
+      accessibilityLabelNextMonth='fakeaccessibilityLabelNextMonth'
+      accessibilityLabelPreviousMonth='fakeaccessibilityLabelPreviousMonth'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      helperText={'Select a publishing date for your pin.'}
+      id="example-helperText"
+      onChange={({value}) => setDate(value)}
+      value={date}
+    />
+
+  )
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="errorMessage"
+    description="
+    DatePicker can display its own error message. To use our errors, simply pass in an errorMessage when there is an error present. Error message overrides helper text.
+  "
+    name="Example: Error message"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(undefined);
+  return (
+    <DatePicker
+      accessibilityLabelNextMonth='fakeaccessibilityLabelNextMonth'
+      accessibilityLabelPreviousMonth='fakeaccessibilityLabelPreviousMonth'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      errorMessage={!date ? "This field can't be blank!" : null}
+      helperText={'https://pinterest.com/'}
```

Comment:
@AlbertCarreras can we update this to be something else? Or leave it empty

Reply:
fixed